### DISCUSSION TITLE
Http status error on starting up neurodesk when running as a container in Docker

### DISCUSSION THREAD

**Discussion by StephWithey** [neurodesk/neurodesk.github.io-hugo#425] [Q&A]
I have been running Neurodesk as a container in Docker with no problems until just over a week ago when the following error popped up on launching Neurodesk locally. An internet search suggests increasing the maxHTTPheadersize in Apache Tomcat which I have tried to do. But the error is still coming up. 
Any suggestions, please? Thanks in advance!
<img width="584" alt="error_neurodesk" src="https://github.com/NeuroDesk/neurodesk.github.io/assets/42540370/abc8005c-e9c5-496c-93d2-31e05801629c">

**Comment by stebo85** [comment]
dear @StephWithey 

that’s a bug that occurs when a Neurodesktop session wasn’t shutdown and crashed. We are working on a solution so that a restart of the neurodeskapp fixes it.

for now, you have 2 ways of fixing it:
1) go into the debug settings of the app and clear the cookie cache

2) Uninstalling the neurodesk app and reinstalling it


@iishiishii that’s the bug that’s needs clearing of the Neurodesk app caches

**Comment by StephWithey** [comment]
@stebo85, thanks so much for the quick reply. I cleared the cache and it is now working again.

**Reply by stebo85** [reply]
Wonderful to hear! Sorry for the bug - we are hoping to have a good fix for the next release:)

**Comment by stebo85** [comment]
this bug is now fixed and the cache is cleared automatically.
